tests/e2e/pageObjects/Viewer.ts

```diff
@@ -0,0 +1,156 @@
+import { Page, expect } from '@playwright/test'
+import { state } from '../hooks'
+import { config } from '../config.js'
+import util from 'util'
+
+export class Viewer {
+  page: Page = state.page
+  elements: Readonly<Record<string, string>> = {
+    userNameSelector: '#oc-login-username',
+    passwordSelector: '#oc-login-password',
+    loginButtonSelector: 'button[type="submit"]',
+    webContentSelector: '#web-content',
+    userMenuButtonSelector: '#_userMenuButton',
+    logoutSelector: '#oc-topbar-account-logout',
+    loginFormSelector: '.oc-login-form',
+    resourceUploadButton: '#upload-menu-btn',
+    fileUploadInput: '#files-file-upload-input',
+    resourceNameSelector: '#files-space-table [data-test-resource-name="%s"]',
+    appbarResourceNameSelector: '#app-top-bar-resource [data-test-resource-name="%s"]',
+    uploadInfoCloseButton: '#close-upload-info-btn',
+    appTopBar: '.oc-app-top-bar .oc-resource', // '.oc-app-top-bar',
+    appTopBarResourceName: '.oc-resource-name',
+    modelViewport: '#preview .model-viewport',
+    modelViewportWrapper: '#preview #scene-wrapper',
+    modelViewportWrapperFullscreen: '#scene-wrapper:fullscreen',
+    modelViewportDescription: '#preview h1.oc-invisible-sr',
+    modelViewportCanvas: '#preview .model-viewport canvas',
+    controlButtonPrev: '.preview-controls-previous',
+    controlButtonNext: '.preview-controls-next',
+    controlButtonFullscreen: '.preview-controls-fullscreen',
+    controlButtonReset: '. preview-controls-reset'
+  }
+
+  async login({ username, password }): Promise<void> {
```

Comment:
let's use static methods and properties. then we can just do `Viewer.login` without instantiating the class
```suggestion
  async static login({ username, password }): Promise<void> {
```

Reply:
@saw-jan if I make the login method static, I get quite a lot of errors...
Do you recommend to use a constructor method for the Ocis pageObject class similar to 
[tests/e2e/pageObjects/OcisPage.js](https://github.com/JankariTech/web-app-presentation-viewer/blob/add-ui-test/tests/e2e/pageObjects/OcisPage.js) in the presentation viewer?

Reply:
My suggestion is to make the whole class static so that we don't have to instantiate it when using

Reply:
@saw-jan when I make everything in this class static I have a problem with the "page"

Reply:
what does the error say?

Reply:
`error TS2304: Cannot find name 'page'.`

Reply:
`error TS2304: Cannot find name 'page'.`

Reply:
Could you make the change to the file? then I can look what might be the problem

Reply:
committed the current state of the code although tests fail. I think it has something to do with the fact that "page" is undefined at the time it is called.

Reply:
I wasn't able to make it work with ocis being a static class